pkg/kubeapiserver/options/authentication.go

```diff
@@ -461,6 +462,19 @@ func (o *BuiltInAuthenticationOptions) ToAuthenticationConfig() (kubeauthenticat
 		if ret.AuthenticationConfig, err = loadAuthenticationConfig(o.AuthenticationConfigFile); err != nil {
 			return kubeauthenticator.Config{}, err
 		}
+		// all known signing algs are allowed when using authentication config
+		// TODO: what we really want to express is 'any alg is fine as long it matches a public key'
+		ret.OIDCSigningAlgs = []string{
+			oidc.RS256,
+			oidc.RS384,
+			oidc.RS512,
+			oidc.ES256,
+			oidc.ES384,
+			oidc.ES512,
+			oidc.PS256,
+			oidc.PS384,
+			oidc.PS512,
+		}
```

Comment:
Should we just have an exported function in `oidc.go` that returns keys from https://github.com/kubernetes/kubernetes/blob/7ec1a89a509906dad9fd6a4635d7bfc157b47790/staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/oidc/oidc.go#L201-L211

Then we don't have to maintain the list in 2 places.

Reply:
Done.